usaspending_api/database_scripts/matview_generator/summary_transaction_month_view.json

```diff
@@ -193,44 +193,43 @@
       "columns": [{"name": "pulled_from", "order": "DESC NULLS LAST"}]
     }, {
       "name": "recipient_unique_id",
-      "where": "recipient_unique_id IS NOT NULL",
```

Comment:
This was removed to ensure we include even NULL DUNS values in the aggregation, correct?

Reply:
The index wasn't being used for the `... WHERE recipient_unique_id IN (.....)` queries if it was a partial index, so I removed it